src/planwise/component/datasets.clj

```diff
@@ -18,13 +19,15 @@
   [record]
   (some-> record
           (update :import-result edn/read-string)
+          (update :import-job edn/read-string)
           (update :mappings edn/read-string)))
 
 (defn dataset->db
   [dataset]
   (some-> dataset
-          (update :import-result pr-str)
-          (update :mappings pr-str)))
+          (update-if-contains :import-result #(some-> % pr-str))
+          (update-if-contains :import-job #(some-> % pr-str))
+          (update-if-contains :mappings #(some-> % pr-str))))
```

Comment:
I understand why this is necessary (the double guard `update-if-contains` and `some->`), but I don't like it much. Any way around it?